nodes/lynx-notification.html

```diff
@@ -1,110 +1,125 @@
 <script type="text/javascript">
     RED.nodes.registerType('lynx-notification', {
         category: 'IoT Open',
-        color: "#12b1be",
+        color: '#12b1be',
         defaults: {
-            server: {value: "", type: "lynx-server"},
-            installation_id: {value: 0, required: true},
-            output_id: {value: 0, required: true},
-            output_name: {value: "", },
-            executor_id: {value: 0, required: true},
-            message_id: {value: 0, required: true},
+            server: { value: '', type: 'lynx-server' },
+            installation_id: { value: 0, required: true },
+            output_id: { value: 0, required: true },
+            output_name: { value: '', },
+            message_id: { value: 0, },
+            executor_id: { value: 0, },
```

Comment:
Fetching the output by ID would give you the message id and executor ID. These two should be possible to remove.